Web failed to build, missing cmake?

Hi, I attempted to build using the default version and it failed, giving this error.
```
==> fast_html
# Sadly, build components separately seems to sporadically fail
cd c_src/lexbor; cmake -DLEXBOR_BUILD_SEPARATELY=OFF -DLEXBOR_BUILD_SHARED=OFF
/bin/sh: cmake: not found
make: *** [Makefile:42: c_src/lexbor/liblexbor_static.a] Error 127
could not compile dependency :fast_html, "mix compile" failed. You can recompile this dependency with "mix deps.compile fast_html", update it with "mix deps.update fast_html" or clean it with "mix deps.clean fast_html"
** (Mix) Could not compile with "make" (exit status: 2).
You need to have gcc and make installed. If you are using
Ubuntu or any other Debian-based system, install the packages
"build-essential". Also install "erlang-dev" package if not
included in your Erlang/OTP version. If you're on Fedora, run
"dnf group install 'Development Tools'".

==> pleroma
The command '/bin/sh -c mix local.rebar --force     && mix local.hex --force     && mix deps.get     && mix compile' returned a non-zero code: 1
```
As I see someone reported about 2 weeks ago (the last issue) a successful build I am going to try stepping back a commit to work out which commit broke this.

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,6 +8,7 @@ ARG PLEROMA_VER=develop
 RUN apk -U upgrade \
     && apk add --no-cache \
     build-base \
+    cmake \
     git
 
 RUN addgroup -g ${GID} pleroma \

